src/css/formulae.scss

```diff
@@ -82,6 +82,10 @@
     align-items: center;
 }
 
+.flex-align-start {
+    align-items: start;
+}
```

Comment:
we have dc__align-start. But yeah it is confusing with text-align classes though

Reply:
Removed this